src/main/java/seedu/address/logic/Messages.java

```diff
@@ -18,7 +18,9 @@ public class Messages {
     public static final String MESSAGE_PERSONS_LISTED_OVERVIEW = "%1$d persons listed!";
     public static final String MESSAGE_DUPLICATE_FIELDS =
                 "Multiple values specified for the following single-valued field(s): ";
-    public static final String MESSAGE_PERSON_NOT_FOUND = "The person with the given identity number was not found";
+    public static final String MESSAGE_UNABLE_TO_EDIT_LOG = "Logs cannot be edited!";
+    public static final String MESSAGE_PERSON_NOT_FOUND = "NRIC: %1$s not found in system, perhaps there was a typo.";
```

Comment:
Can consider using alternative words for typo since it's an internet slang. If can't find clear and concise alternative, is alright.